docker/Dockerfile

```diff
@@ -5,18 +5,20 @@ RUN apt-get update && \
     apt-get install -y python3-pip && \
     pip3 install --upgrade awscli
 
+COPY docker/package.json ./
+RUN npm install
+
 WORKDIR /usr/src/app
 
-COPY docker/package.json .
-RUN npm install
+# copy our code and create build folder in container
+COPY public ./public
+COPY img ./img
+COPY build ./build
 
-# Bundle app source
-COPY build/ ./build
 COPY docker/server ./server
-COPY start.sh .
+COPY docker/start.sh ./
```

Comment:
Does the `start.sh` file still exist?

Reply:
it's not needed anymore. We simplified so that the Docker runs its only command internally.